tendrl/integrations/gluster/check_cluster_status.py

```diff
@@ -26,6 +27,25 @@ def run():
                     "unhealthy",
                     prevExist=False
                 )
+
+                cluster_tendrl_context = NS.tendrl.objects.ClusterTendrlContext(
+                    integration_id=int_id
+                ).load()
+
+
+                msg = "Cluster {0} has went to unhealthy state".format(
```

Comment:
Cluster <name> moved to unhealthy state?

Reply:
ack